binding/go/README.md

```diff
@@ -40,7 +37,7 @@ Create an instance of the engine and transcribe an audio file:
 ```go
 import . "github.com/Picovoice/leopard/binding/go"
 
-leopard = Leopard{AccessKey: "${ACCESS_KEY}"}
+leopard = NewLeopard("${ACCESS_KEY}")
 err := leopard.Init()
 if err != nil {
     // handle err init
```

Comment:
can you edit below so we show there is transcript and word + make sure to change main readme file

Reply:
I just noticed in Go we could have always made process return `transcript, words, err`